source/docs/networking/networking-introduction/ip-configurations.rst

```diff
@@ -39,13 +39,13 @@ The Field Network runs a DHCP server with pools for each team that will hand our
 On the Field Static Configuration
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-It is also possible to configure static IPs on your devices to accommodate devices or software which do not support mDNS. When doing so you want to make sure to avoid addresses that will be in use when the robot is on the field network. These addresses are ``10.TE.AM.1`` and ``10.TE.AM.4`` for the OpenMesh radio and the field access point and anything ``10.TE.AM.20`` and up which may be assigned to a device still configured for DHCP. The roboRIO network configuration can be set from the webdashboard.
+It is also possible to configure static IPs on your devices to accommodate devices or software which do not support mDNS. When doing so you want to make sure to avoid addresses that will be in use when the robot is on the field network. These addresses are ``10.TE.AM.1`` for the OpenMesh radio, ``10.TE.AM.4`` for the field router, and anything greater than ``10.TE.AM.20`` which may be assigned to a device configured for DHCP or else reserved. The roboRIO network configuration can be set from the webdashboard.
 
 -  OpenMesh radio - Static ``10.TE.AM.1`` programmed by Kiosk
 -  roboRIO - Static ``10.TE.AM.2`` would be a reasonable choice, subnet mask
    of ``255.255.255.0`` (default)
 -  Driver Station - Static ``10.TE.AM.5`` would be a reasonable choice,
-   subnet mask **must** be ``255.0.0.0``
+   subnet mask **must** be ``255.0.0.0`` to enable the DS to reach both the robot and FMS Server
```

Comment:
Let's add a "without additional configuration" so folks don't get confused why DHCP works.

Reply:
Agreed and if we're going to do this then I'd suggest being up front about the alternative, which is to configure the default gateway.

Reply:
Works for me